Feat/routing (#55)
BearStudio/start-ui-web-docs: pages/tutorials/create-your-first-crud.mdx

});
```

### Create the routes.ts file

> To prevent raw string urls all over our files, we are creating a `routes.ts` file at the root of the feature folder.

Reviewer: ```suggestion
To prevent raw string urls all over our files, we are creating a `routes.ts` file in the `src/features/projects` folder.
```
To stay coherent with the zod schema documentation above, and because when you say the root of the feature folder, it could be 'features/routes.ts" ( or at least we could say the feature's folder so it's less ambiguous )

Author: Thanks
